### DISCUSSION TITLE
Drawing an half circle

### DISCUSSION THREAD

**Discussion by s-celles** [py-pdf/pypdf#703] [Q&A]
Hello,

I'm looking for a way to draw an half circle (ie an arc of circle with angle from 0 to 180 degrees)
Unfortunately I haven't find a method for that purpose. 
So I'm wondering if I missed it or if that's not currently implemented. 
If that's the case (ie it's not currently implemented) I wonder how it could be done. Ideally I think an arc or sector method could be added with parameters such as xcenter, ycenter, radius, start_angle,  stop_angle.

Kind regards

**Comment by MartinThoma** [comment]
Did you find a solution?
